icu4j/main/classes/core/src/com/ibm/icu/number/NumberFormatterSettings.java

```diff
@@ -486,6 +492,44 @@ public T scale(Scale scale) {
         return create(KEY_SCALE, scale);
     }
 
+     /**
+     * Sets the `usage` of the unit.
+     * <p>
+     * NOTE: `usage` will change the output unit depending on the `Locale`
+     * and the unit value.
+     *    For Example:
+     *        Locale: en_US
+     *        Usage : length-person
+     *        Unit  : Meter
+     *
+     *        If the unit value is 0.25, the output will be "10 inches."
+     *        If the unit value is 1.50, the output will be
+     *                                           "4 feet and 11 inches"
+     * </p>
+     *
+     * <P>
+     * If the input usage is not exist (e.g. length-dinosaur) or is
+     * misspelled, the output unit **will not change**.
+     * </p>
+     *
+     * <p>
+     * Pass an element of string to this setter.
+     * For example:
+     *
+     * <pre>
+     * NumberFormatter.with().usage("length-person")
```

Comment:
I think this API text is not up-to-date with the latest version in C++: #1289

Reply:
This means I need to update the proposal too ?

Reply:
We can change the API docs as long as the API signature is the same.

Reply:
done